Update kinesis docs (#14768)
apache/druid: docs/development/extensions-core/kinesis-ingestion.md

### AWS authentication

Druid uses AWS access and secret keys to authenticate Kinesis API requests. There are a few ways to provide this information to Druid:

> 1. Using roles or short-term credentials:

Reviewer: ```suggestion
1. Using roles or short-term credentials. 
```
Lines 435-438 do not align with line 433. Consider indenting the first line of the paragraph to align it with line 433. The same suggestion applies to lines 440 and 442.

Author: thanks, done!

Druid looks for credentials set in [environment variables](https://docs.aws.amazon.com/cli/latest/userguide/cli-configure-envvars.html),
via [Web Identity Token](https://docs.aws.amazon.com/IAM/latest/UserGuide/id_roles_providers_oidc.html), in the
default [profile configuration file](https://docs.aws.amazon.com/cli/latest/userguide/cli-configure-files.html), and from the
EC2 instance profile provider (in this order).

2. Using long-term security credentials:

You can directly provide your AWS access key and AWS secret key in the `common.runtime.properties` file as shown in the example below:

```properties
druid.kinesis.accessKey=AKIAWxxxxxxxxxx4NCKS
druid.kinesis.secretKey=Jbytxxxxxxxxxxx2+555
```

> Note: AWS does not recommend providing long-term security credentials in configuration files since it might pose a security risk.

> If this approach is used, it takes precedence over all other methods of providing credentials.

Reviewer: ```suggestion
If you use this approach, it takes precedence over all other methods of providing credentials.
```
